lib/big_brother/health_fetcher.rb

```diff
@@ -17,9 +17,14 @@ def self.interpol_status(interpol_nodes, fwmark)
       end
 
       response = _first_interpol_response(urls)
-      response.response_header.status == 200 ? JSON.parse(response.response) : []
-    rescue JSON::ParserError
-      []
+      return [] unless response
+      if response.response_header.status == 200
```

Comment:
there's no case where this will not be true, given we only set result when it's a 200